.env.sample

```diff
@@ -1,6 +0,0 @@
-# Smarty streets secrets
-SMARTY_AUTH_ID=
-SMARTY_AUTH_TOKEN=
-
-# Okta secrets
-OKTA_API_KEY=
```

Comment:
What was the reasoning for removing this file? We do reference it [in the README](https://github.com/CDCgov/prime-simplereport#developing-locally), as these env vars are actually needed to run the app locally IIRC.

Reply:
Ah, this was a mistake as I renamed it to the sample for cypress. I'll add this back!